CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Fix incorrect save button behavior in Circulation Rules Editor. Refs UICIRC-296.
 * Add handling of the special item Any for intermediate menus. Refs UICIRC-334.
 * Add full hierarchial path to location codes that are used in the input field of the circulation rules editor. Refs UICIRC-333.
+* Clean up Loan history model before sending to backend. Refs UITEN-57.
```

Comment:
It seems, some date will be cleaned even without sending to backend. Am I right?